addons/assignGear/functions/fnc_assignGearSupplyBox.sqf

```diff
@@ -40,46 +41,35 @@ if (!isClass _path) exitWith {
     diag_log text format ["[POTATO-assignGear] - No loadout found for %1 (typeOf %2)", _theBox, typeOf _theBox];
 };
 
-//Clean out starting inventory (even if there is no class)
-clearWeaponCargoGlobal _theBox;
-clearMagazineCargoGlobal _theBox;
-clearItemCargoGlobal _theBox;
-clearBackpackCargoGlobal _theBox;
-
-private _transportMagazines = getArray(_path >> "TransportMagazines");
-private _transportItems = getArray(_path >> "TransportItems");
-private _transportWeapons = getArray(_path >> "TransportWeapons");
-private _transportBackpacks = getArray(_path >> "TransportBackpacks");
-
-// transportMagazines
-{
-    (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
-    _theBox addMagazineCargoGlobal [_classname, parseNumber _amount];
-    nil
-} count _transportMagazines; // count used here for speed, make sure nil is above this line
-
-// transportItems
-{
-    (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
-    _theBox addItemCargoGlobal [_classname, parseNumber _amount];
-    nil
-} count _transportItems; // count used here for speed, make sure nil is above this line
-
-// transportWeapons
-{
-    (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
-    private _disposableName = [cba_disposable_LoadedLaunchers, _classname, "get", ""] call FUNC(getDisposableInfo);
-    if (_disposableName != "") then {
-        TRACE_2("cba_disposable_LoadedLaunchers replace",_classname,_disposableName);
-        _classname = _disposableName;
+private _subBoxes = "true" configClasses _path;
+if (_subBoxes isNotEqualTo []) then {
+    private _boxName = getText (_path >> "boxCustomName");
+    if (_boxName isNotEqualTo "") then {
+        _theBox setVariable ["ace_cargo_customName", _boxName, true];
```

Comment:
`QACEGVAR(cargo,customName)`, see other comment for details

Reply:
I think it'd be better to use the ace function to set the box name. The reason it is what it is was because I based it off of your code from #539 here https://github.com/BourbonWarfare/POTATO/blob/6ddca1d4e4095d477d43f9d2d0a0386f07721e10/addons/assignGear/functions/fnc_assignGearVehicle.sqf#L366

Reply:
like I said, I am a hypocrite :p 

I'll probably do a pass of POTATO at some point to fix all of these, change if you desire it's non-blocking